src/game/client/components/console.cpp

```diff
@@ -285,7 +285,7 @@ void CGameConsole::CInstance::OnInput(IInput::CEvent Event)
 				if(m_Type == CONSOLETYPE_LOCAL || m_pGameConsole->Client()->RconAuthed())
 				{
 					char *pEntry = m_History.Allocate(m_Input.GetLength() + 1);
-					mem_copy(pEntry, m_Input.GetString(), m_Input.GetLength() + 1);
+					str_copy(pEntry, m_Input.GetString(), m_Input.GetLength());
```

Comment:
```suggestion
					str_copy(pEntry, m_Input.GetString(), m_Input.GetLength() + 1);
```

Should be the size of the destination buffer, which is what we allocate above.